src/compiler/scala/tools/nsc/typechecker/ContextErrors.scala

```diff
@@ -1175,6 +1172,9 @@ trait ContextErrors {
       def MissingParameterOrValTypeError(vparam: Tree) =
         issueNormalTypeError(vparam, "missing parameter type")
 
+      def ParentSealedInheritanceError(parent: Tree, psym: Symbol) =
+        NormalTypeError(parent, "illegal inheritance from sealed " + psym )
+
```

Comment:
It's moved from `ContextErrors` to `NamerContextErrors` so that it's accessible in `Namers`.